### PR TITLE
Feature/simple simple report

### PR DESCRIPTION
**Pull request by rDr4g0n**
This PR makes a new template, "simpler", that is based on "simple" (it is separate to avoid introducing any breaking changes). The goal is to generate an html file that can be displayed anywhere, without dependencies. This is a requirement for displaying in jenkins using the html publisher plugin. This *should* address issue #101 (I cannot verify in my jenkins environment until this PR is merged). **[edit] works in jenkins!**

It also removes the implicit dependency on moment.js for friendly time formatting of the last build time. This would close issue #98, however it does remove that particular feature (humanizing the time is easy, but parsing the timestamp is not so reliable).

### CODE REVIEW

**Comment by rDr4g0n** [comment]
The first 2 commits just remove external dependencies. The rest of the commits change the style of the report to appear more compact and distinctive.

acceptance1

**Comment by Frankusky** [comment]
Would be nice if the error message is displayed

**Comment by rDr4g0n** [comment]
@Frankusky A good idea, but I wanted to keep it in line with the "simple" report which apparently doesnt include that.

**Comment by gkushang** [comment]
@rDr4g0n thanks for the PR, and fixing the open issues!

The `simpler` theme is quite similar to `simple`. To not have the overhead of maintaining two similar themes, I'd suggest you to go ahead and make the similar changes to `simple` theme. It won''t be the breaking change, and can keep one theme. The background colors would highlight the Pass/Fail in simple theme. 

Please feel free to add `error` message to the simple theme.

```
            <% if (step.result.error_message) { %>
            <div id="error">
                <pre class=info><%= step.result.error_message.replace(/</g, '(').replace(/>/g, ')') %></pre>
            </div>
            <% } %>
```

Let me know what you think about it.

Cheers!

File: templates/simple/script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        this.innerText = "Screenshot -";
+        this.nextElementSibling.style.display = "block";
+    }
+}
+
+var toggleEls = document.querySelectorAll("a.toggle");
+
+for(var i = 0; i < toggleEls.length; i++){
+    toggleEls[i].addEventListener("click", toggleScreenshot);
+}
```
Comment (gkushang) [inline templates/simple/script.js]: Can you also split the [condition](https://github.com/gkushang/cucumber-html-reporter/blob/develop/lib/reporter.js#L132) to use this particular `script.js` while generating the report?

Comment (rDr4g0n) [inline templates/simple/script.js]: Yeah, that was a bug in my change that i missed. The below commit fixes it. Im kinda in the middle of changes, so dont merge yet!

**Comment by gkushang** [comment]
@rDr4g0n can you please take a look at the conflicts?

**Comment by rDr4g0n** [comment]
@gkushang yep, next up on my list

**Comment by rDr4g0n** [comment]
simpler2

**Comment by rDr4g0n** [comment]
@gkushang OK, im good with these changes. If you wanna go ahead and review and/or merge, have at it!

**Comment by rDr4g0n** [comment]
I just tried using the reporter after develop was merged in and am getting an error. I suspect I just need to update some configs or something since it looks like some major changes were made, but just in case it is helpful:

```
TypeError: Cannot read property 'type' of undefined
    at /home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:215:48
    at Array.forEach (native)
    at /home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:213:41
    at Array.forEach (native)
    at /home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:211:31
    at Array.forEach (native)
    at /home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:198:30
    at Array.forEach (native)
    at setStats (/home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:181:23)
    at generateReport (/home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:373:13)
    at Object.generate (/home/jay/src/thing-web/node_modules/cucumber-html-reporter/lib/reporter.js:419:9)
    at Object.generateReport [as generate] (/home/jay/src/thing-web/node_modules/cucumber-html-reporter/index.js:30:21)
    at Object.after (/home/jay/src/thing-web/test/e2e/nightwatch.globals.js:50:18)
    at CliRunner.runGlobalAfter (/home/jay/src/thing-web/node_modules/nightwatch/lib/runner/cli/clirunner.js:496:19)
    at /home/jay/src/thing-web/node_modules/nightwatch/lib/runner/cli/clirunner.js:402:16
    at _fulfilled (/home/jay/src/thing-web/node_modules/nightwatch/node_modules/q/q.js:834:54)
```

**Comment by gkushang** [comment]
Can you do a clean install? The _develop_ now has Cucumber 3 support, and I suspect you might want to install Cucumber V3

File: templates/simple/features.tmpl [hunk 2]
```diff
@@ -13,17 +13,19 @@
...
-                   <a class="screenshot" href="<%= step.image %>">
-                      <img class="screenshot" style="height:100%;width:100%" id="my_images" src="<%= step.image %>"/>
-                   </a>
+                   <div class="step-screenshot-wrap">
+                     <a class="toggle" href="#">Screenshot -</a>
+                     <img class="screenshot" src="<%= step.image %>"/>
+                   </div>
+             <% } %>
+             <% if (step.result.error_message) { %>
+                   <pre class="step-error-message"><%= step.result.error_message.replace(/</g, '(').replace(/>/g, ')') %></pre>
```
Comment (gkushang) [inline templates/simple/features.tmpl]: 👍

**Comment by gkushang** [comment]
published with V3.0.1

**Pull request merged**